### ISSUE TITLE
config files corresponds to models in detection model zoo  

### ISSUE THREAD

**Issue by jinxin0924** [tensorflow/models#2074]
[Tensorflow detection model zoo] (https://github.com/tensorflow/models/blob/master/object_detection/g3doc/detection_model_zoo.md) said:
> In the table below, we list each such pre-trained model including:
a model name that corresponds to a config file that was used to train this model in the samples/configs directory.

But I can not find any model config files about COCO dataset.  

Did I miss something?

**Comment by sleepfin** [comment]
I find that default total steps in `faster_rcnn_resnet101_voc07.config` is 800k and initial learning rate is 0.0001. But the total steps is 80k and the initial learning rate is 0.01 according to the paper (vgg16+faster-rcnn).  Any explaination for this?

**Comment by derekjchow** [comment]
Use the *_pets.config files as a basis for coco model configs, changing num_classes from 37 to 90.
